fn/operate/logging.md

```diff
@@ -13,16 +13,7 @@ All logs are emitted in [logfmt](https://godoc.org/github.com/kr/logfmt) format
 ## Call ID
 
 Every function call/request is assigned a `call_id`. If you search your logs, you can track all the activity
```

Comment:
`call_id` is no longer in the logs.  Each line contains timestamp, app id, and function id. Here's a sample:

`2018-11-21T13:50:28-05:00 linuxkit-025000000001 app_id=01CV2SYZ9NNG8G00RZJ0000001,fn_id=01CWVSNBD0NG8G00RZJ00000 Caused by: java.lang.RuntimeException: Something went horribly wrong!`

Reply:
Otherwise LGTM!

Reply:
Fixed with the second tweak.